policyengine_us/parameters/gov/states/co/ssa/co_state_supplement/age_range.yaml

```diff
@@ -0,0 +1,19 @@
+description: Age range defining someone eligible for Colorado SSI State Supplement
+brackets:
```

Comment:
This is a new way of defining age eligibility! I think this could especially make sense for the childless EITC, which has two nonzero endpoints. Not sure we need it here though, @nikhilwoodruff wdyt?

Reply:
No this is great here- we actually also use this format elsewhere already, in Medicaid eligibility formulas. The PolicyEngine policy page is clunky with scales UI-wise but that's an issue for the UI and not the parameter here.